dlk/python/dlk/templates/include/global.tpl.h

```diff
@@ -129,6 +129,7 @@ struct Base<QuantizedPacked<T>> {
 #define MAX_SIZE_IM2COL_INPUTS_PER_LAYER {{ params.max_size_im2col_inputs_per_layer }}
 #define MAX_SIZE_IM2COL_QINPUTS_PER_LAYER {{ params.max_size_im2col_qinputs_per_layer }}
```

Comment:
It looks possible to remove this definition of `MAX_SIZE_IM2COL_QINPUTS_PER_LAYER`. Accordingly, it also looks possible to remove the computation of `params.max_size_im2col_qinputs_per_layer`.